scripts/add-evita.py

```diff
@@ -42,7 +42,11 @@ def parse_evita_raps(node):
 		    node.update({'attr': dict()})
 
 		attr = node.get('attr')
-
+		
+		if len(evita_line) != 10:
+			print ("EVITA:: tag should have exactly 9 elements in attack vector node", node,"where it only has",len(evita_line)-1)
```

Comment:
nice check, thanks.

can you please use the `%` operator for consistency in the repo? e.g. "EVITA:: tag should ... vector node %s where it " % (node, ...)